src/components/Input/index.tsx

```diff
@@ -32,6 +33,8 @@ const Input = forwardRef(function InputComponent(props: InputProps, ref) {
 
 Input.displayName = ComponentNames.Input;
 
+Input.defaultProps = defaultInputProps;
```

Comment:
Feels very confusing to me to have both `Input.defaultProps` and `Input.DEFAULT_PROPS`.
If I remember correctly, `defaultProps` are passed to the validation function. But, looks like they won't have the default validation function in there, so people can't extend the default validation like they did before.
Let's talk about this.

Reply:
I was planning on exporting each of the default validation functions instead of adding them to `DEFAULT_PROPS`, especially because the validation function is no longer a prop for the `Input` and other components.

So it doesn't make sense for it to be in the `DEFAULT_PROPS` anymore. The only reason it still exists is for backwards compatibility